### DISCUSSION TITLE
170 upgrade and so-status show so-suricata as missing

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#8871] [Unsupported Versions]
Hi all

We have upgraded our distributed deployment to 170 from 160, and it all seemed to go well but both of our sensor nodes are showing a "Fault" state, and on each of them a so-status shows that so-suricata is "MISSING".  
We have done a salt highstate for the sensors, and it has come back with no errors, but suricata never starts, even after rebooting the node.
Can someone point me where I might start debugging this issue?

Thanks
Ross

**Comment by reyesj2** [comment]
Check soup.log for errors in upgrade
```
sudo cat /root/soup.log
``` 

I have had success with running
```
sudo docker restart $(docker ps -qa)
```

What is the output of 
```
sudo docker images | grep suricata
```
Check the logs of suricata for clues 

```
sudo cat /opt/so/log/suricata/suricata.log
```
